content/en/about/contribute/github/index.md

```diff
@@ -115,6 +117,34 @@ See this
 [README](https://github.com/istio/istio.io/blob/master/README.md) for
 instructions.
 
+## Branching
+
+We use multiple branches to track documentation for different versions of Istio.
+The master branch is where active doc work tables place, this is where you should
+generally be making changes.
+
+On the day of an Istio release, a release branch is created from master for that
+release. For example, there are branches called `release-1.0`, `release-1.1`,
+`release-1.2` and so forth.
+
+The `istio.io` public site is produced by processing content from the current
+release branch. The `preliminary.istio.io` site is produced by processing
+content from the master branch. And the `archive.istio.io` site is
+produced by processing content from all prior release branches.
+
+Given how branching works, if you submit a change into the master branch,
+that change will not appear on `istio.io` until the next major Istio release
+happens. If your documentation change is relevant to the current Istio release,
+then it's probably worth also applying your change to the current release branch.
+You can do this easily and automatically by using the special cherry-pick labels
+on your documentation PR. For example, if you introduce a correction in a PR to
+the master branch, you can apply the `cherrypick/release-1.4` label in order to
+merge this change to the `release-1.4` branch.
```

Comment:
Should we also mention that the label will create a second PR, and the author may need to add an `@googlebot I consent.` comment to satisfy the CLA and let it merge?

Reply:
Done.